### DISCUSSION TITLE
3 instances of Weather?

### DISCUSSION THREAD

**Discussion by doctorwizz** [valinet/ExplorerPatcher#739] [General]
Weather did not show up after boot from cold start. It was working and it is still checked.
Edit: I clicked "Show Weather on the taskbar" again. I thought it would uncheck it. The weather appeared again. I click it again. And a second instance of weather appeared. A third click and now have 3 weathers on the taskbar. I know it's development version. So it's cool. Just reporting my findings.

**Comment by doctorwizz** [comment]
I rebooted again and there were no weather widgets on the taskbar. I clicked the show weather on taskbar and one widget appeared.

**Reply by valinet** [reply]
So is it working or not? Does the option toggle them or does it add multiple ones, one every time you click?

**Reply by doctorwizz** [reply]
Still adds one for every click.

**Reply by valinet** [reply]
Can you delete `C:\Program Files\ExplorerPatcher\WebView2Loader.dll` and restart Explorer? The People button should show instead of the weather widget. Then, use the same option in Properties to toggle it ("Show Weather on the taskbar") and tell me whether it creates multiple People buttons. After you are done, restore `C:\Program Files\ExplorerPatcher\WebView2Loader.dll` (easiest is to execute `ep_setup.exe` again). Thanks.

**Reply by doctorwizz** [reply]
I dont see the Peoples button after restarting explorer. But every time I click show weather it creates a new peoples button

**Reply by valinet** [reply]
Okay, then it's not a problem regarding weather, it's some bug in `explorer` that fails to save the state of the feature, apparently, and then it itself gets confused (thinks the button is not there, so it creates one). Yeah, I don't know, the setting is stored as DWORD in: `HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced\People\PeopleBand`. Restart `explorer`, have it show no people button, use the option to enable it and then sign out and then sign back in and check whether the mentioned registry entry has a value of 1, indicating that it saved the setting correctly and it knows it has to create the People button. And generally, yeah, idk what fix to suggest, idk why it happens in the first place.
